<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>WengoPhone</string>
  </property>
  <property name="toolTip" >
   <string/>
  </property>
  <property name="whatsThis" >
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>0</number>
    </property>
    <item row="2" column="0" >
     <widget class="QTabWidget" name="tabWidget" >
      <widget class="QWidget" name="tabContactList" >
       <attribute name="title" >
        <string>Contacts</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabCallHistory" >
       <attribute name="title" >
        <string>History</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabDialpad" >
       <attribute name="title" >
        <string>Dialpad</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabPhoneCall" >
       <attribute name="title" >
        <string>Call</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabHome" >
       <attribute name="title" >
        <string>Home</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item row="1" column="0" >
     <widget class="QFrame" name="statusbar" >
      <property name="frameShape" >
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0" >
     <widget class="QFrame" name="iconbar" >
      <property name="frameShape" >
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="4" >
        <widget class="QPushButton" name="pushButton_4" >
         <property name="text" >
          <string/>
         </property>
         <property name="icon" >
          <iconset resource="wengophone.qrc" >:/pics/iconbar_search.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="flat" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="3" >
        <widget class="QPushButton" name="pushButton_3" >
         <property name="text" >
          <string/>
         </property>
         <property name="icon" >
          <iconset resource="wengophone.qrc" >:/pics/iconbar_chat.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="flat" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2" >
        <widget class="QPushButton" name="pushButton_2" >
         <property name="text" >
          <string/>
         </property>
         <property name="icon" >
          <iconset resource="wengophone.qrc" >:/pics/iconbar_cal.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="flat" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QPushButton" name="pushButton" >
         <property name="text" >
          <string/>
         </property>
         <property name="icon" >
          <iconset resource="wengophone.qrc" >:/pics/iconbar_sendfile.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="flat" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="5" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="0" >
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="callButton" >
        <property name="icon" >
         <iconset resource="wengophone.qrc" >:/pics/call_button.png</iconset>
        </property>
        <property name="iconSize" >
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="flat" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="hangUpButton" >
        <property name="icon" >
         <iconset resource="wengophone.qrc" >:/pics/hangup_button.png</iconset>
        </property>
        <property name="iconSize" >
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="flat" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="5" column="0" >
     <widget class="QStackedWidget" name="phoneLineStackedWidget" >
      <property name="sizePolicy" >
       <sizepolicy>
        <hsizetype>5</hsizetype>
        <vsizetype>0</vsizetype>
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex" >
       <number>0</number>
      </property>
      <widget class="QWidget" name="page" />
     </widget>
    </item>
    <item row="3" column="0" >
     <widget class="QComboBox" name="phoneComboBox" >
      <property name="sizePolicy" >
       <sizepolicy>
        <hsizetype>7</hsizetype>
        <vsizetype>0</vsizetype>
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="editable" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>Help</string>
    </property>
    <addaction name="actionShowForum" />
    <addaction name="actionShowHelp" />
    <addaction name="actionShowAbout" />
   </widget>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>Wengo</string>
    </property>
    <addaction name="actionShowWengoAccount" />
    <addaction name="actionOpenWengoAccount" />
    <addaction name="actionEditMyProfile" />
    <addaction name="separator" />
    <addaction name="actionExit" />
   </widget>
   <widget class="QMenu" name="menuContacts" >
    <property name="title" >
     <string>Contacts</string>
    </property>
    <addaction name="actionAddContact" />
    <addaction name="actionSetLogin" />
   </widget>
   <widget class="QMenu" name="menuTools" >
    <property name="title" >
     <string>Tools</string>
    </property>
    <addaction name="actionCreateConferenceCall" />
    <addaction name="actionSendSms" />
    <addaction name="separator" />
    <addaction name="actionConfiguration" />
    <addaction name="actionAdvancedConfiguration" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuContacts" />
   <addaction name="menuTools" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>524</y>
     <width>271</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <action name="actionShowWengoAccount" >
   <property name="text" >
    <string>View My Wengo Account</string>
   </property>
  </action>
  <action name="actionOpenWengoAccount" >
   <property name="text" >
    <string>Open Another Wengo Account</string>
   </property>
  </action>
  <action name="actionExit" >
   <property name="text" >
    <string>Exit</string>
   </property>
  </action>
  <action name="actionShowForum" >
   <property name="text" >
    <string>Forum Access</string>
   </property>
  </action>
  <action name="actionShowHelp" >
   <property name="text" >
    <string>Help</string>
   </property>
  </action>
  <action name="actionShowAbout" >
   <property name="text" >
    <string>About</string>
   </property>
  </action>
  <action name="actionAddContact" >
   <property name="text" >
    <string>Add Contact</string>
   </property>
  </action>
  <action name="actionConfiguration" >
   <property name="text" >
    <string>Configuration</string>
   </property>
  </action>
  <action name="actionSendSms" >
   <property name="text" >
    <string>Send SMS</string>
   </property>
  </action>
  <action name="actionSetLogin" >
   <property name="text" >
    <string>Set login</string>
   </property>
  </action>
  <action name="actionAdvancedConfiguration" >
   <property name="text" >
    <string>Advanced Configuration</string>
   </property>
  </action>
  <action name="actionEditMyProfile" >
   <property name="text" >
    <string>&amp;Edit my profile</string>
   </property>
  </action>
  <action name="actionCreateConferenceCall" >
   <property name="text" >
    <string>Create Conference Call</string>
   </property>
  </action>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources>
  <include location="wengophone.qrc" />
 </resources>
 <connections/>
</ui>
